### PR TITLE
Added inference time measure.

### PR DESCRIPTION
**Pull request by SalahAdDin**
This kind of measure are so important in order to compare this thecnique with others.

### CODE REVIEW

File: keras_retinanet/utils/eval.py [hunk 1]
```diff
@@ -96,7 +100,7 @@ def _get_detections(generator, model, score_threshold=0.05, max_detections=100,
         scores_sort = np.argsort(-scores)[:max_detections]
 
         # select detections
-        image_boxes      = boxes[0, indices[scores_sort], :]
+        image_boxes      =  boxes[0, indices[scores_sort], :]
```
Comment (hgaiser) [inline keras_retinanet/utils/eval.py]: Could you remove this extra space?

File: keras_retinanet/utils/eval.py [hunk 2]
```diff
@@ -232,4 +238,6 @@ def evaluate(
         average_precision  = _compute_ap(recall, precision)
         average_precisions[label] = average_precision, num_annotations
 
-    return average_precisions
+        inference_time = np.sum(all_inferences)/generator.size()
```
Comment (hgaiser) [inline keras_retinanet/utils/eval.py]: Add spaces around operators (`/`) as per PEP8.

File: keras_retinanet/utils/eval.py [hunk 3]
```diff
@@ -165,7 +171,7 @@ def evaluate(
         A dict mapping class names to mAP scores.
     """
     # gather all detections and annotations
-    all_detections     = _get_detections(generator, model, score_threshold=score_threshold, max_detections=max_detections, save_path=save_path)
+    all_detections, all_inferences     = _get_detections(generator, model, score_threshold=score_threshold, max_detections=max_detections, save_path=save_path)
```
Comment (hgaiser) [inline keras_retinanet/utils/eval.py]: Since this no longer aligns with the code below, could you remove the extra spaces before the `=` sign?

**Comment by SalahAdDin** [comment]
Ok, done.

**Comment by hgaiser** [comment]
Thanks :+1:

**Pull request merged**